### PR TITLE
Cleanup unit test for gmlanutils.py

### PR DESCRIPTION
**Pull request by polybassa**
Another cleanup. This removes the use of ISOTPSoftSockets within the unit tests.

### CODE REVIEW

**Comment by polybassa** [comment]
@gpotter2 This is ready for merge

File: test/p0f.uts [hunk 1]
```diff
@@ -73,7 +73,7 @@ assert fingerprint_mtu(pkt) == "Ethernet or modem"
 + Tests for p0f_impersonate
 
 = Check that the impersonated packet is properly detected by p0f
-~ netaccess
+~ netaccess disabled
```
Comment (gpotter2) [inline test/p0f.uts:76]: Please don't throw this in here

**Comment by polybassa** [comment]
No worries. I just wanted to check if all my test are working. I planned to remove this after the review.

On September 11, 2021 7:04:37 PM GMT+02:00, gpotter2 ***@***.***> wrote:
***@***.*** commented on this pull request.

**Comment by polybassa** [comment]
This is ready... finally

**Comment by polybassa** [comment]
@gpotter2 Can this PR get merged, please.

This PR introduces TestSocket objects for gmlan utils tests. This changes reduce the complexity of the unit tests, speed up the execution and improve the stability of these tests.

**Review by guedou — APPROVED** [review]

**Comment by polybassa** [comment]
@guedou Thanks!

**Pull request merged**